Tools/CISupport/ews-build/steps.py

```diff
@@ -147,9 +140,9 @@ def add_patch_id_url(self):
             self.addURL('Patch {}'.format(patch_id), Bugzilla.patch_url(patch_id))
 
     def add_pr_details(self):
-        pr_number = self.get_pull_request_number()
+        pr_number = self.getProperty('github.number')
         if pr_number:
-            self.addURL('Pull request {}'.format(pr_number), self.pr_url(pr_number=pr_number))
+            self.addURL('Pull request {}'.format(pr_number), GitHub.pr_url(self.getProperty('repository'), pr_number) or '')
```

Comment:
still unclear who is setting this 'repository' property. Can we just skip it altogether?

Reply:
'repository' comes from the GitHub hook, Making it optional may be reasonable, but we don't want to drop it entirely. This is one of the arguments that will change for security pull-requests.